src/components/generics/vacation-test-forms/employeeVacationRequests.tsx

```diff
@@ -9,23 +9,24 @@ import { CalendarPickerView } from "@mui/x-date-pickers";
 import strings from "localization/strings";
 import DateRangePicker from "../date-range-picker/date-range-picker";
 import { FilterScopes } from "types";
+import { VacationRequestStatus, VacationType } from "generated/client";
 
 /**
  * interface type request
  */
 interface Request {
   request: {
     id: number;
-    vacationType: string;
-    comment: string;
+    vacationType: VacationType | string;
+    message: string;
     employee: string;
     days: number;
-    startDate: string;
-    endDate: string;
+    startDate: Date | string;
+    endDate: Date | string;
     remainingDays: number;
-    status: string;
-    created: string;
-    updated: string;
+    status: VacationRequestStatus | string;
+    created: Date | string;
+    updated: Date | string;
```

Comment:
I assume there is a reason for these union types of Date | string, can it not just be Date?